kitsune/messages/jinja2/messages/includes/macros.html

```diff
@@ -52,27 +52,27 @@
 
 {% macro outbox_message(message) -%}
   <span class="to">
-  {% if message.recipient %}
+  {% if message.to %}
     <p><strong>{{ _('To') }}:</strong>
-      {% if message.recipients > 1 -%}
-        {% set comma = joiner(', ') %}
-        {% for user in message.to.all() -%}
+    {% set comma = joiner(', ') %}
+      {% for user in message.to.all() -%}
           {{ comma() }}
           {{ name_link(user) }}
-        {%- endfor %}
       {% else %}
         {{ name_link(message.recipient) }}
-      {% endif %}
+      {% endfor %}
```

Comment:
Outdent all of these lines -- the `for...else...endfor` to match the line with the `{% set comma = joiner(', ') %}`.

Reply:
I did as requested - I need to find some rules/a linter for Jinja files.....